client/src/utils/bulkUpdateBatchQuantity.js

```diff
@@ -0,0 +1,34 @@
+import { getBatchItem, updateBatchItem } from '../services/items';
+
```

Comment:
Ah yes, thanks for raising this - I was meaning to consolidate some of those into a single file since their functionality is related - I'll do that, along with some documentation to explain why and what they do :)